themes/bootstrap3/js/common.js

```diff
@@ -240,19 +240,31 @@ var VuFind = (function VuFind() {
   };
 
   var loadHtml = function loadHtml(_element, url, data, success) {
-    var $elem = $(_element);
-    if ($elem.length === 0) {
+    var element = document.querySelector(_element);
+  
+    if (!element) {
       return;
     }
-    $.get(url, typeof data !== 'undefined' ? data : {}, function onComplete(responseText, textStatus, jqXhr) {
-      if ('success' === textStatus || 'notmodified' === textStatus) {
-        $elem.html(updateCspNonce(responseText));
-      }
-      if (typeof success !== 'undefined') {
-        success(responseText, textStatus, jqXhr);
+  
+    var xhr = new XMLHttpRequest();
+    xhr.open('GET', url, true);
+  
+    xhr.onload = () => {
+      if (xhr.status >= 200 && xhr.status < 300 || xhr.status === 304) {
+        element.innerHTML = updateCspNonce(xhr.responseText);
+  
+        if (typeof success !== 'undefined') {
+          success(xhr.responseText, xhr.statusText, xhr);
+        }
       }
-    });
-  };
+    };
+  
+    xhr.onerror = () => {
+      console.error('Request failed');
```

Comment:
I'm not sure this adds any useful information. But maybe we should update the target element's content to `VuFind.translate('error_occurred')` to indicate to the user that something went wront.